### PR TITLE
Change behaviour of FullWidth property in Fields

### PR DESCRIPTION
**Pull request by carlosforero**
* Rename TextField.fullWidth to TextField.block
* Add FullWidth property in TextField and SelectField,
  in order to expand the width to 100%

### CODE REVIEW

File: src/js/SelectFields/SelectField.js [hunk 1]
```diff
@@ -581,7 +588,9 @@ export default class SelectField extends Component {
     const menuProps = {
       isOpen: open,
       close: this.close,
-      className: classnames('md-select-field-menu-container', menuClassName),
+      className: classnames('md-select-field-menu-container', menuClassName, {
+        'full-width': {fullWidth}
```
Comment (mlaursen) [inline src/js/SelectFields/SelectField.js]: this line should just be `'full-width': fullWidth` The way it is now is always truthy

File: src/scss/components/_text-fields.scss [hunk 2]
```diff
@@ -35,6 +35,16 @@
       width: 100%;
     }
   }
+
+  &.full-width {
+    width: 100%;
+    .md-text-field {
```
Comment (mlaursen) [inline src/scss/components/_text-fields.scss]: Lines 41-46 can be merged with lines 91-93 which will also solve the scss lint error

**Comment by carlosforero** [comment]
@mlaursen I didn't see your comments. Let me fix it.

**Comment by carlosforero** [comment]
@mlaursen everything is ok now.

**Comment by mlaursen** [comment]
Looks good. I'll merge tomorrow morning

**Pull request merged**